### PR TITLE
Added MutableString, MutableNumber and MutableBool types

### PR DESCRIPTION
**Pull request by wouterdb**

### CODE REVIEW

File: model/_init.cf [hunk 1]
```diff
@@ -477,3 +477,54 @@ end
 implement AgentConfig using std::none
 
 index AgentConfig(agentname)
+
+
+entity MutableString:
+"""
+Wrapper for string values, used to pass a string out of an if statement::
```
Comment (bartv) [inline model/_init.cf:484]: It is also useful for creating modifiable lists. Maybe for documentation purposes create a muteable type parent and add the docs there.

Comment (wouterdb) [reply model/_init.cf:484]: Could you create an example of the modifiable list scenario? 

I have to admit that the docs have a lot of repetition, but it does have the advantage that for each entity, the docs are 0 clicks away from the ting itself.

Comment (bartv) [reply model/_init.cf]: I added the example. I would link to the parent in the docs of the childeren. But you may also expand my example to the other types. Your choice

**Comment by wouterdb** [comment]
merged in manually via 978f2c8f94fa0c5f921cbbe6bf8740725a8e2076

**Pull request closed without merging**